Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
             targets: ["Confidence"])
     ],
     dependencies: [
-        .package(url: "git@github.com:open-feature/swift-sdk.git", from: "0.1.0"),
```

Comment:
Interestingly, builds from main started failing since the dependency is "from" and not "exact". Maybe ok to keep it like this, forces us to keep things up to date :)